cmp/grid/GridModel.js

```diff
@@ -1085,9 +1102,20 @@ export class GridModel {
         };
     }
 
+    parseChooserModel(chooserModel) {
+        if (XH.isMobileApp) {
+            return chooserModel ? this.markManaged(new MobileColChooserModel(this)) : null;
+        }
+
+        if (chooserModel instanceof ColChooserModel) {
```

Comment:
think we should remove this section here, as discussed above -- but if we were to leave it, this code incorrectly recreating a new model on line 1111